sktime/forecasting/trend/_pwl_trend_forecaster.py

```diff
@@ -51,6 +51,15 @@ class ProphetPiecewiseLinearTrendForecaster(_ProphetAdapter):
         automatic changepoint selection. Large values will allow many
         changepoints, small values will allow few changepoints.
         Recommended to take values within [0.001,0.5].
+    yearly_seasonality: str or bool or int, default="auto"
+        Fit yearly seasonality.
```

Comment:
bit unclear, I would say sth like "whether to fit yearly seasonality, and if yes how" and then explain values one by one, what they do